frontend/components/auction/surplus/SurplusAuctionsTable.vue

```diff
@@ -76,7 +76,7 @@
 <script lang="ts">
 import Vue, { PropType } from 'vue';
 import { Table } from 'ant-design-vue';
-import { SurplusAuctionTransaction } from 'auctions-core/src/types';
+import { SurplusAuction, SurplusAuctionTransaction } from 'auctions-core/src/types';
```

Comment:
We used the wrong type in this file. Please remove the import of the `SurplusAuction` and fix the type (from SurplusAuction to SurplusAuctionTransaction) in this line:

https://github.com/sidestream-tech/unified-auctions-ui/blob/130fbb7e3b808954802cd635b160b9084dd9eced/frontend/components/auction/surplus/SurplusAuctionsTable.vue#L228